lib/index.js

```diff
@@ -48,41 +77,133 @@ function compile_template(template_name, type) {
+                }
+            },
+            Subject: { /* required */
+                Data: person.subject,
+                /* required */
+                Charset: 'utf8'
+            }
+        },
+        Source: person.senderEmailAddress || process.env.SENDER_EMAIL_ADDRESS,
+        /* required */
+        ReplyToAddresses: [
+            person.replyToAddress || process.env.SENDER_EMAIL_ADDRESS
+        ]
+    };
+    ses.sendEmail(params, function(err, data) {
+        if (err) { console.log('SES ERROR:', err, err.stack); } // an error occurred
+        return callback(err, data);
+    });
+
+}
+
+
+/**
+ * sendemail method takes a template name and person array object for the To's, CC's & BCC and uses
+ * AWS SES to send the desired email.
+ * @param {String} template_name - the template to use for the email
+ * @param {Object} template_objects - the objects that will be replaced in template dynamically {{token}}
+ * @param {String} subject - the subject of the email
+ * @param {Object} person - the object containing the details of the
+ * person to whom we want to send the email. Requires both name
+ * and email. if you don't *know* the name of the person, leave it
+ * an empty string.
+ * @param {Function} callback - continuation function called after
+ * the email has been sent.
+ */
+function sendemails(template_name, template_objects, subject, personTO, personCC, personBCC, callback) {
+    // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
+
+    var toEmailAddress = [];
+    if (personTO === null || typeof personTO === 'undefined') {
+        console.log("Email addresses in TO's cannot be resolved");
+    } else {
+        for (var l = 0; l < personTO.length; l++) {
+            toEmailAddress.push(personTO[l].email);
+        }
+    }
+
+    var ccEmailAddress = [];
+    if (personCC === null || typeof personCC === 'undefined') {
+        console.log("Email addresses in cc cannot be resolved");
+    } else {
+        for (var k = 0; k < personCC.length; k++) {
+            ccEmailAddress.push(personCC[k].email);
+        }
+    }
+
+    var bccEmailAddress = [];
+    if (personBCC === null || typeof personBCC === 'undefined') {
+        console.log("Email address in the BCC's cannot be resolved");
+    } else {
+        for (var j = 0; j < personBCC.length; j++) {
+            bccEmailAddress.push(personBCC[j].email);
         }
-      },
-      Subject: { /* required */
-        Data: person.subject, /* required */
-        Charset: 'utf8'
-      }
-    },
-    Source: person.senderEmailAddress || process.env.SENDER_EMAIL_ADDRESS, /* required */
-    ReplyToAddresses: [
-      person.replyToAddress || process.env.SENDER_EMAIL_ADDRESS
-    ]
-  };
-  ses.sendEmail(params, function(err, data) {
-    if (err) {console.log('SES ERROR:', err, err.stack);} // an error occurred
-    return callback(err, data);
-  });
+    }
+
+    var params = {
+        Destination: {
```

Comment:
this is not DRY ... https://en.wikipedia.org/wiki/Don't_repeat_yourself 
We are repeating _large_ chunks of code here instead of simply re-using the `sendemail` function... this adds maintenance.